### DISCUSSION TITLE
r/city-china-jiangsu-changzhou-liyang-yihaogonglu-01

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#49] [Announcements]
# r/city-china-jiangsu-changzhou-liyang-yihaogonglu-01

游记｜溧阳1号公路，三色轨迹，一路风光

https://blog.ops-coffee.cn/r/city-china-jiangsu-changzhou-liyang-yihaogonglu-01

**Comment by 2443266417** [comment]
每当看到这种游记文。都是一种享受

**Reply by ops-coffee** [reply]
很高兴你能喜欢

**Comment by beyond1314** [comment]
写的很详细，不错啊，我从V2转过来看你的游记，今天看了一天了，溧阳这个有机会要去自驾看看的。
不过也推荐下台州1号公路，温州168公路，俗称的沿着G228海岸线。

**Reply by ops-coffee** [reply]
哈哈，安排，后边都要去

**Comment by chonghua** [comment]
北线自然风光好，山比较多，最近几年我也是到处自驾，爬爬山，山顶露个营，溧阳土著
